docs/readme-windows.md

```diff
@@ -0,0 +1,41 @@
+# Building on Windows
+
+We recommend using the Visual Studio toolchain because it is easier for development on Windows. However, if you wish to use MinGW, follow the instructions on [this page](readme-mingw.md).
```

Comment:
```suggestion
We recommend using the Visual Studio toolchain because it is easier for development on Windows. However, if you wish to use MinGW or CodeBlocks, follow the instructions on [this page](readme-mingw.md).
```